runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -98,6 +98,111 @@ function M.on_refresh(err, _, ctx, _)
   return vim.NIL
 end
 
+--- @class vim.lsp.inlay_hint.get.opts
+--- @field bufnr number?
```

Comment:
Nit, but I would use `integer` for buffer numbers and client IDs.

Reply:
i keep forgetting that exists (typescript brain lol), thanks

Reply:
lol tell me about it